ember-accounts/app/mixins/analytics.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+//A temporary solution to get GA shipped, Really what I want here
+//is conventionally queued Analytics events - @discorick
+export default Ember.Mixin.create({
+  actions: {
+    purchase(model) {
+      window.analytics.page({
+        url: `/settings/${this.get('model.login')}/upgrade`,
```

Comment:
Technically shouldn't `url` include the full path? Not sure if that matters for GA integration. ([Docs](https://segment.com/docs/libraries/analytics.js/#page))